go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/planetscale/psdb
 go 1.19
 
 require (
-	github.com/bufbuild/connect-go v1.3.0
+	github.com/bufbuild/connect-go v1.5.1
```

Comment:
:eyes:
been awhile eh? lol

Reply:
Well, this is more kept up to date in edge, where this is consumed, and go.mod is a "min version" policy. So it doesn't particularly matter unless you were consuming this library directly, which... maybe soon-ish!

Reply:
ohhh I see